data/csrc/anemone/anemone_mempool.c

```diff
@@ -0,0 +1 @@
+../../../lib/anemone/csrc/anemone_mempool.c
```

Comment:
no, I just think it's cleaner than putting this path in Preamble -- happy to just do that